### PR TITLE
Implemented dynamic wait for redeployment of Che, if was suspended/idling

### PR DESCRIPTION
**Pull request by mlabuda**

### CODE REVIEW

**Comment by ibuziuk** [comment]
[test]

File: src/main/java/io/fabric8/che/starter/openshift/CheDeploymentConfig.java [hunk 1]
```diff
@@ -40,49 +41,83 @@
 	@Value("${che.openshift.start.timeout}")
 	private String startTimeout;
 
-	public void startPodIfNeeded(OpenShiftClient client) {
+	public void deployCheIfSuspended(OpenShiftClient client) {
 		final ClientDeployableScalableResource<DeploymentConfig, DoneableDeploymentConfig> deploymentConfig = client
 				.deploymentConfigs().inNamespace(project).withName(deploymentConfigName);
```
Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/openshift/CheDeploymentConfig.java]: should it just call getDeploymentConfig(client) ?

Comment (mlabuda) [inline src/main/java/io/fabric8/che/starter/openshift/CheDeploymentConfig.java]: yes, thanks. I missed this in refactoring.

**Comment by ibuziuk** [comment]
@mlabuda thanks - pushed to master

**Pull request merged**